.travis.yml

```diff
@@ -19,7 +21,7 @@ matrix:
     - env: TOXENV=py37
       python: nightly
     - env: TOXENV=pypy
-      python: pypy
+      python: pypy-5.4
```

Comment:
@dstufft Nope. https://travis-ci.org/pradyunsg/pip/jobs/249185227

There's `pypy3.5-5.8.0` which failed because of some directory was not being resolved properly... https://travis-ci.org/pradyunsg/pip/jobs/249044310